plugins/out_stackdriver/stackdriver.h

```diff
@@ -86,6 +86,10 @@
 #define FLB_STACKDRIVER_FAILED_REQUESTS      1001   /* failed requests */
 #endif
 
+#define GRPC_STATUS_CODES_SIZE 32
```

Comment:
[Optional] I wonder if we need a comment here, or a link to https://grpc.github.io/grpc/core/md_doc_statuscodes.html …